src/modules/core/components/PermissionsLabel/permissions.ts

```diff
@@ -7,30 +7,55 @@ const MSG = defineMessages({
     id: 'core.Permission.rootLabel',
     defaultMessage: 'Root',
   },
+  rootInfoMessage: {
+    id: 'core.Permission.rootInfoMessage',
```

Comment:
These are `PermissionsLabel`, not `Permission`:
```suggestion
    id: 'core.PermissionsLabel.rootInfoMessage',
```

Please make sure to update all of them